.github/workflows/check-standalone-bayesian-optimization.yml

```
@@ -0,0 +1,31 @@
+name: Check instantiation of bayesian optimization on python 3.8
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install -r requirements-gpsearchers.txt --upgrade pip
+        python -m pip install typing_extensions
+    - name: Check that stand-alone bayesian-optimation example runs
+      run: |
+        PYTHONPATH='./' python examples/launch_standalone_bayesian_optimization.py
```

Awesome, this is much cleaner, small nit the following is more standard AFAIK:
```suggestion
        PYTHONPATH=. python examples/launch_standalone_bayesian_optimization.py
```